images-to-azure-storage-account/README.md

```diff
@@ -111,6 +110,9 @@ To delete the deployed Azure services, including all images in the storage accou
 az group delete --name <resource group name>
 ```
 
+The Azure API Management instance will be [soft-deleted](https://aka.ms/apimsoftdelete) and remain known to Azure for 24 hours. This will prevent you from redeploying the application to a resource group with the same name. Please select a new name for the resource group, wait 24 hours or [purge the instance using the Azure REST API](https://aka.ms/apimsoftdelete#purge-a-soft-deleted-instance).
```

Comment:
I might have said wrong previously, but the correct value should be 48 hours according to https://docs.microsoft.com/azure/api-management/soft-delete#soft-delete-behavior